test/refactoring/rename/basic.swift

```diff
@@ -46,6 +54,8 @@ print(top)
 // RUN: diff -u %S/Outputs/basic/local.swift.expected %t.result/local.swift
 // RUN: %refactor -rename -source-filename %s -pos=20:7 -new-name 'bottom' > %t.result/top_level.swift
 // RUN: diff -u %S/Outputs/basic/top_level.swift.expected %t.result/top_level.swift
+// RUN: %refactor -rename -source-filename %s -pos=26:12 -new-name new_SP1 > %t.result/SP1.swift
+// RUN: diff -u %S/Outputs/basic/SP1.swift.expected %t.result/SP1.swift
```

Comment:
Could you also add an equivalent `RUN` line for the `-find-local-rename-ranges` refactoring? It's almost the same as the `-rename` refactoring, but it returns just the ranges to be renamed, without worrying about what they should be replaced with. It will probably need a similar fix to what you've done for `-rename`. There's an example of that one two lines below this one for reference.